crates/core/tedge_mapper/src/core/converter.rs

```diff
@@ -2,13 +2,11 @@ use crate::c8y::dynamic_discovery::DiscoverOp;
 use async_trait::async_trait;
 use mqtt_channel::Message;
 use mqtt_channel::Topic;
-use mqtt_channel::TopicFilter;
 use std::fmt::Display;
 use tracing::error;
 
 #[derive(Debug)]
 pub struct MapperConfig {
-    pub in_topic_filter: TopicFilter,
```

Comment:
I believe the `MapperConfig` is the generic struct to define each mapper specific characteristics. `in_topic_filter` is one of the identity as a mapper. Why can we remove it from here?

Reply:
Removed it as the mapper actors are connected to the mqtt actor via actor APIs only and not using this mapper abstraction. My plan is to eventually get rid of this mapper abstraction itself.

Reply:
Even the `out_topic` doesn't really make much sense as mappers like the c8y mapper has more than one topic to which it sends its outputs.